CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Unreleased
 
+## New features
+
+### Replace back link placeholder URLs with javascript
+
+Previously, users had to manually replace the `href` for each Back link in the page templates.
+
+Now, we use JavaScript to make the Back link take users to the previous page by default.
+
+This approach is not appropriate for production scenarios, where you cannot rely on JavaScript being available at all times.
+
+Users can still override the `href` attribute if they need different behaviour.
```

Comment:
Probably worth asking @EoinShaughnessy to take a look at this on Monday. I think in release notes we normally talk to users as if they're reading this, e.g: "you can still override" vs "users can still override", but probably worth checking with Eoin.

Reply:
Thanks @vanitabarrett! @domoscargin I added a few suggestions :)